tests/test_docstrings.py

```diff
@@ -0,0 +1,22 @@
+"""Basic tests"""
+import os.path
+import sys
+
+from sphinx_pytest.plugin import CreateDoctree
+
+
+def test_multiline_docstring_d212(sphinx_doctree_no_tr: CreateDoctree, snapshot):
+    """Basic test for molde with multiline string with D212 format"""
```

Comment:
"molde" is probably a forgotten local reference?

Reply:
Rather a typo, fixed now